### DISCUSSION TITLE
../blocks/download/node_modules/@joint/plus/joint-plus.mjs  Can't import the named export 'V' from non EcmaScript module (only default export is available)

### DISCUSSION THREAD

**Discussion by Hemal-LPT218** [clientIO/joint#2714] [Q&A]
### Introduction

After installing the JointJS+ library using file installation, errors occur when running the project after importing the JointJS+ library.
![error after import jointjs](https://github.com/user-attachments/assets/a300791d-149e-4671-81a4-3dccd38a689e)

### Steps to reproduce

"react": "16.11.0",
"react-dom": "16.11.0",
"react-scripts": "3.1.1",
"@types/node": "13.13.52",
 "@types/react": "16.9.11",
 "typescript": "4.1.3"
 
This version was in my project. After installing the JointJS+ library, I created a demo as shown in the React example and ran the project but got an error.

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by MartinKanera** [comment]
I am having trouble reproducing this. If you are trying to include JointJS+ in an already established project, I suggest updating the `react-scripts` to `5.0.1`. In case you are starting with the project, it might be a good idea to use React with Vite just like [JointjS+ React Tutorial](https://docs.jointjs.com/learn/integration/react#create-a-react-project) suggests to ensure the latest dependencies for React development.

**Reply by Hemal-LPT218** [reply]
@MartinKanera 
I have an already existing project. After successfully installing react-scripts version 5.0.1, I encountered errors at project runtime (as shown in the image). Additionally, I'm using Node v16.20.2, npm 8.19.4, and Yarn 1.22.19.
![after react script upgred ](https://github.com/user-attachments/assets/bb0ef329-fe00-41f1-a111-0fe3ef9bfcc7)

**Reply by kumilingus** [reply]
See https://stackoverflow.com/questions/51185690/uncaught-typeerror-cannot-read-property-oneof-of-undefined.

**Reply by MartinKanera** [reply]
I found a similar issue on [StackOverflow](https://stackoverflow.com/a/75542707), adding the `"props-types": "15.8.1"` as a project dependency seemed to work for someone. However, I cannot confirm it since I didn't manage to reproduce the error you are encountering. It would be helpful if you could provide me with some simple git repository where this problem occurs if you require any further assistance.
